weblate/templates/component.html

```diff
@@ -96,6 +96,7 @@
       <li><a href="{% url 'manage-access' project=object.project.slug %}">{% trans "Users" %}</a></li>
       {% endif %}
       {% if user_can_edit_component %}
+      <li><a href="{% url 'create-component' %}#existing">{% trans "Duplicate Component" %}</li>
```

Comment:
I'd expect the current component to be selected when clicking on the link here.

Reply:
Okay, To have `initial` value for `component` field in `get_form`, where I can set `initial` of `form` in `fetch_params` from the request?

```
form.fields['component'].initial = form.initial.get('component')
```